Week07/lesson14/classwork/responsive_articles_with_menu/css/styles.css

```diff
@@ -0,0 +1,216 @@
+/* Box Model Hack */
+* {
+ box-sizing: border-box;
+}
+
+/* Clearfix hack */
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after{
+  clear: both;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  font-size: 13px;
+  background-color: #F6F0E4;
+}
+
+/* Basic/Mobile Article Styling: */
+
+nav {
+  background-color: #7BAFA4;
+}
+
+nav article {
+  padding: 0 20px;
+}
+
+menu {
+  float: right;
+}
+
+.menubutton {
+  position: relative;
+  background: white;
+  height: 5px;
+  width: 30px;
+  border-radius: 9px;
+  margin-bottom: 2px;
+}
+
+ul {
+  padding: 0;
+  margin: 0 20px;
+  transition: max-height 0.5s;
+  max-height: 0px;
+  overflow: hidden;
+}
+
+.visible {
+  max-height: 606px;
+}
+
+li {
+  padding: 10px;
+  border-bottom: 1px solid #F6F0E4;
+}
+
+li:first-child {
+  border-top: 1px solid #F6F0E4;
+}
+
+li:last-child {
+  margin-bottom: 20px;
+}
+
+li:hover, li:focus, menu:hover > div, menu:focus > div {
+  background-color: #488A7C;
+}
+
+a, a:visited {
+  text-decoration: none;
+  color: white;
+}
+
+main {
+  margin: 0 auto;
+}
+
+article {
+  padding: 20px;
+}
+
+h2 {
+  font-family: 'PT Serif', serif;
+  font-size: 2em;
+  margin: 20px 0;
+}
+
+p {
+  margin: 0;
+}
+
+img {
+  display: block;
+  width: 100%;
+}
+
+/* Responsive Article Styling: */
+
+@media (min-width: 480px) { /* mobile-landscape */
+  /* limit <main> to 440px and do necessary styling to match screenshot
+  Use classes and elements present in HTML as clues */
+  main {
+    width: 440px;
+    font-size: 0px;
```

Comment:
Don't think it works in the same way if I bring it down to the <article> level as I actually need the space between two articles to be 0 so need to specify within <article>'s parent which is main?